framework/sandstone.cpp

```diff
@@ -3792,6 +3777,29 @@ int main(int argc, char **argv)
         }
     }
 
+    if (list_tests_action) {
+        list_tests(list_tests_action);
+        return EXIT_SUCCESS;
+    }
+
+#ifndef __OPTIMIZE__
+    logging_printf(LOG_LEVEL_VERBOSE(1), "THIS IS AN UNOPTIMIZED BUILD: DON'T TRUST TEST TIMING!\n");
+#endif
+
+    load_cpu_info(sApp->enabled_cpus);
+    setup_signals();
+
+    init_shmem(UseSharedMemory);
+    debug_init_global(on_hang_arg, on_crash_arg);
+    pthread_setcancelstate(PTHREAD_CANCEL_DISABLE, nullptr);
+
+    print_application_banner();
+    logging_init_global();
+    cpu_specific_init();
+    random_global_init(seed);
+    background_scan_init();
+
```

Comment:
Nitpick: unnecessary extra line. In fact, remove both and let the SSL init be merged into this section.